Bikeini/screens/AddBike.js

```diff
@@ -586,67 +427,90 @@ class AddBike extends React.Component {
               </TouchableOpacity>
             </View>
             <View style={styles.radio}>
-              <RadioGroup
-                horizontal="true"
-                radioButtons={radios.type}
-                onPress={(data) => { this.radioUpdater(data, 'type'); }}
-                flexDirection="row"
+              <ButtonGroup
+                selectedButtonStyle={{ backgroundColor: '#44ccad' }}
+                onPress={(data) => { this.setBikeData('type', data ? 'FOUND' : 'STOLEN'); }}
+                selectedIndex={bikeData.type === 'STOLEN' ? 0 : 1}
+                buttons={['Stolen', 'Found']}
```

Comment:
This is so much better than the long list that existed before! Much clearer code now!